index.html

```diff
@@ -17,7 +17,7 @@
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, user-scalable=yes" />
   <title>QFieldCloud - Seamless fieldwork for QGIS</title>
-  <meta name="description" content="QFieldCloud - Seamless fieldwork for QGIS" />
+  <meta name="description" content="QFieldCloud allows to synchronize and merge the data collected by your team in QField. From small individual projects to large data collection campaigns, QFieldCloud removes the pain from synchronizing and merging data." />
```

Comment:
Could we add the keyword "collaborate" (e.g. by replacing on eof the two "synchronize" terms?

Reply:
@m-kuhn , I'll replace the first one, collaborate is a nicer word than sync.